mathesar_ui/src/systems/record-selector/RecordSelectorWindow.svelte

```diff
@@ -105,15 +93,12 @@
 
 <style>
   .record-selector-window {
-    --z-index__record-selector__divider: 1;
-    --z-index__record_selector__row-header: 2;
-    --z-index__record_selector__thead: 3;
-    --z-index__record_selector__focused-input-with-overflow: 4;
-    --z-index__record_selector__thead-row-header: 5;
-    --z-index__record_selector__focused-input: 6;
-    --z-index__record_selector__shadow-inset: 7;
-    --z-index__record_selector__overlay: 8;
-    --z-index__record_selector__above-overlay: 9;
+    --z-index__record_selector__row-header: 1;
+    --z-index__record_selector__thead: 2;
+    --z-index__record_selector__thead-row-header: 3;
+    --z-index__record_selector__shadow-inset: 4;
+    --z-index__record_selector__overlay: 5;
+    --z-index__record_selector__above-overlay: 6;
```

Comment:
should we add a note for this convention in the styling guide? I still don't think I understand this notation fully.

Reply:
I added https://github.com/centerofci/mathesar-wiki/commit/07d21b7805beca4e4884ef23828ed33abe373628 to explain this convention more. Does this make sense?